src/bitcoind.ts

```diff
@@ -9,12 +150,12 @@ export const getBalancesDetail = async (): Promise<
   const balances: { wallet: string; balance: number }[] = []
 
   for await (const wallet of wallets) {
-    // do not use the default wallet for now (expect for testing).
-    if (wallet === "") {
+    // do not consider the "outside" wallet in tests
+    if (wallet === "outside") {
```

Comment:
Can we remove this conditional that that is only related to testing? Would be better to have no test-related code int he production code.

Reply:
This bypass is in there to be able to test the getBalance function, which is used to check the balances after each test. Maybe it can be abstracted into a "tests bypass" function, but having only a single wallet to bypass it seems ok to me for now.